tests/unit/Swift/Transport/SendmailTransportTest.php

```diff
@@ -33,7 +33,7 @@ public function testCommandCanBeSetAndFetched()
         $this->assertEquals('/usr/sbin/sendmail -oi -t', $sendmail->getCommand());
     }
 
-    public function testSendingMessageIn_t_ModeUsesSimplePipe()
+    public function testSendingMessageInTModeUsesSimplePipe()
```

Comment:
We had some of those cases in an internal project as well. But I think we can live with it.